serde-api/src/main/java/io/micronaut/serde/annotation/SerdeImport.java

```diff
@@ -41,7 +42,16 @@
      *
      * @return The type to enable serialization for.
      */
-    Class<?> value();
+    Class<?> value() default void.class;
+
+    /**
+     * Specifies a package name where types can be found. Only publicaly accessible types will be processed.
```

Comment:
I assume nested package are not processed. It would be good to specify it in the javadoc.